topics/imaging/tutorials/multiplex-tissue-imaging-TMA/tutorial.md

```diff
@@ -0,0 +1,421 @@
+
+
+The raw files for each round (10 in total) of the exemplar-002 data are available on [cancer.usegalaxy.org](https://cancer.usegalaxy.org) under **Data Libraries** (Figure 2.). Import the raw files into a new history as a **list collection**.
+
+
+![getData](../../images/multiplex-tissue-imaging-TMA/ex2_getData.png "Finding the Exemplar-002 data on cancer.usegalaxy.org Data Libraries.")
+
+
+> ### {% icon hands_on %} Hands-on: Data import to history
+>
+> 1. Create a new history for this tutorial
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. Import the files for Exemplar-002 from the Shared Data Library to the new history
+>
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+{: .hands_on}
+
+> ### {% icon warning %} **Imaging platform differences**
+> 
+> The Exemplar-002 raw images are in *ome.tiff* format; however, commonly seen raw file-types are *ome.tiff*, *tiff*, *czi*, and *svs*. If your input images are not *ome.tiff* or *tiff*, you may have to edit the dataset attributes in Galaxy to allow tools to recognize them as viable inputs. 
+>
+{: .warning}
+
+
+# Tile illumination correction with **BaSiC Illumination**
+
+Commonly, raw MTI data will consist of one image per round of imaging. These individual round images are frequently captured in tiles, and there can be slight variations in how each tile was illuminated across the course of imaging. Prior to tile stitching and image registration, the tiles have to undergo illumination correction with **BaSiC Illumination** ({% cite Peng2017 %}) to account for this. Unlike many of the other tools in this workflow, BaSiC has no extra parameters to think about: Just input the collection of raw images and press *go*!
+
+Two new list collections will appear in the history upon completion: 
+
+  - BaSiC Illumination on Collection `X`: FFP (flat-field)
+  - BaSiC Illumination on Collection `X`: DFP (deep-field)
+
+> ### {% icon hands_on %} Hands-on: Illumination correction
+>
+> 1. {% tool [BaSiC Illumination](basic_illumination) %} with the following parameters:
+>
+>    - {% icon param-collection %} *"Raw Cycle Images: "*: List collection of raw images
+>
+{: .hands_on}
+
+
+# Stitching and registration with **ASHLAR**
+
+After illumination is corrected across round tiles, the tiles must be stitched together, and subsequently, each round mosaic must be registered together into a single pyramidal OME-TIFF file. **ASHLAR** from MCMICRO provides both of these functions. 
+
+> ### {% icon comment %} Important detail: Marker File
```

Comment:
It's not clear for me where we obtain this markers file from? I found it eventually in the data library, but it won't be clear to folks trying to run this tutorial